test/unit/mongo/MongoTripodTestBase.php

```diff
@@ -95,6 +95,14 @@ protected function setUp()
         $configFileName = dirname(__FILE__).'/data/config.json';
 
         $config = json_decode(file_get_contents($configFileName), true);
+        if(getenv('TRIPOD_DATASOURCE_RS1_CONFIG'))
```

Comment:
Where does this env var get set? On the build server? Does it need documenting anywhere?